velox/core/QueryConfig.h

```diff
@@ -48,17 +48,10 @@ class QueryConfig {
 
   explicit QueryConfig(std::unordered_map<std::string, std::string>&& values);
 
-  static constexpr const char* kCodegenEnabled = "codegen.enabled";
```

Comment:
This property is still used in Prestissimo. We'd need it removed from there first before we can merge this change.

```
bcode/github/presto-trunk/presto-native-execution/presto_cpp/main/common/Configs.cpp:673:34: error: no member named 'kCodegenEnabled' in 'facebook::velox::core::QueryConfig'
          BOOL_PROP(QueryConfig::kCodegenEnabled, c.codegenEnabled()),
                    ~~~~~~~~~~~~~^
```

Perhaps, we can land all the changes except QueryConfig.h first.

Reply:
Thats ok, should I leave a todo there? ( Since it would be a bit weird )

Reply:
I added these back and put them under #ifdef VELOX_ENABLE_BACKWARD_COMPATIBILITY .

This way, next time Prestissimo updates Velox version, it will have to remove usage of these configs. Once that happens we'll be able to remove them.

I'm working on merging this PR.

Thanks.

Reply:
Thanks! Would you push this directly? Or let me add a `VELOX_ENABLE_BACKWARD_COMPATIBILITY` condition?

Reply:
Oh this is got merged

Reply:
Yes, I updated directly. Thank you for the contribution.